crev-lib/src/proofdb.rs

```diff
@@ -205,8 +206,10 @@ pub struct ProofDB {
     /// who -(trusts)-> whom
     trust_id_to_id: HashMap<Id, HashMap<Id, TimestampedTrustLevel>>,
 
-    url_by_id: HashMap<Id, TimestampedUrl>,
-    url_by_id_secondary: HashMap<Id, TimestampedUrl>,
+    verified_url_by_id: HashMap<Id, TimestampedUrl>,
+
+    /// Untrusted
+    unverified_url_by_id: HashMap<Id, TimestampedUrl>,
```

Comment:
I would rename it to `url_by_id_reported_by_others`.

And I'd change it to be `HashMap<Id, HashMap<Id, TimestampedUrl>>`, so we can potentially filter for urls reported by most trusted user available or something.

Reply:
For reverse mappings I'd add `url_by_id_reported_source` or something. Whatever you think is necessary to enable some additional heuristics that you might need.

Reply:
https://github.com/crev-dev/cargo-crev/pull/299#discussion_r405189411

Reply:
Using WoT for picking the most likely URL is a good idea. I would need help implementing it, because I'm not familiar with the `TrustSet`, etc.